docs/profiling/numpy_calculator.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
```

Comment:
[nit] can we add a comment block to state that this and the following classes are for profiling. To use the service then refer to the correct classes.

Reply:
Like, for people consuming the package? These don't get installed at all for them at all.

Reply:
It's more that someone can see this code and copy paste it as the 'correct' code. It would be nice to have a warning there (even if we can't force anyone to read it).

Reply:
Ok, added a comment block to the profiling classes to only use them for profiling.